src/query/config/src/config.rs

```diff
@@ -1254,6 +1242,18 @@ pub struct QueryConfig {
 
     // ----- the following options/args are all deprecated               ----
     // ----- and turned into Option<T>, to help user migrate the configs ----
+    /// OBSOLETED: The maximum memory size of the buffered data collected per insert before being inserted.
+    #[clap(long)]
```

Comment:
No need to be OBSOLETED, only remove them.
As we know, there is no user used them.